libs/design/src/atoms/form/input/input.component.scss

```diff
@@ -1,12 +1,12 @@
-@import 'daffodil-util';
+@import '../../../scss/daff-util';
 
 .daff-input {
     height: inherit;
     background: $white;
     border: 1px solid $white;
     padding: 15px;
     margin: 0;
-    font-size: $global-font-size;
+    font-size: 1rem;
```

Comment:
Shouldn't this be $global-font-size?

Reply:
So, assuming the body is set to $global-font-size` we don't need the variable here.

Reply:
I would think that would work, but there's a 1.14285714rem font-size on input[text] elements. I don't know where it's coming from.

Reply:
That's coming from foundation-sites. Since we're not planning on relying on foundation anymore, we should consider getting rid of the library. I tried removing it from the `styles.scss` and it wasn't pretty. This would be another PR.